src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -321,6 +322,8 @@ private void initFragmentsWithFile() {
                     startTextPreview(file);
             }
 
+            switchLayout(getFile());
```

Comment:
There is a local variable ```file``` set to ```getFile()``` 13 lines above; I would use it instead of calling ```getFile()``` again.